make-it/EmMakefile

```diff
@@ -74,11 +74,6 @@ ifeq ($(BUILD), profile)
    EM_OPTFLAG = --profiling
 endif
 
-ifneq ($(wildcard custom.mak),)
```

Comment:
Sorry, may be, I was not clear. We should only remove the custom.mak file from source control. The references to it in the Makefile's should stay as-is before this commit. It is an internal build tool convention @spathiwa used in the make files. custom.mak is used locally by developers to override flags and build with that configuration as default. So, the convention is to use custom.mak the way it is used in Makefiles but not check in the file in source control but add the path of the custom.mak files to .gitignore so that it does not annoy the developers who use that. I broke that convention misremembering the usage by checking in the file. This change and the change below in make-it/Makefile should continue to reference custom.mak as before. Sorry for the confusion I created on this to begin with.